ui/k-info-panel/switch_info.kytos

```diff
@@ -61,18 +116,101 @@
        window.d3.json(endpoint, function(error, result) {
          self.flows = result[dpid].flows
        })
-     }
+     },
+
+     rowClicked (link) {
+       var subtitle = ""
+       if(link.metadata.link_name === undefined){
```

Comment:
@Alopalao operators probably won't set an empty string, but if it's set it would display an empty string, could you check for non truthy values here?

Reply:
Added the metadata table and changed link_name checker in commit [a14cfb5065aa2aaa3ac638107dbf810a0bd03b3e](https://github.com/kytos-ng/topology/pull/73/commits/a14cfb5065aa2aaa3ac638107dbf810a0bd03b3e)

Reply:
@Alopalao, yeah, since the metadata is just a key value pair storage, it's desirable to show the empty string as they are, so no transformation needed.